fossid-webapp-client/src/main/kotlin/api/identification/common/Component.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.fossid.api.identification.common
+
+data class Component(
+    val copyright: String?,
+    val cpe: String?,
+    val id: Long?,
+
+    val includeInReport: Int?,
+
+    val copyleft: Int?, // 0 or 1
```

Comment:
This probably should be a `Boolean`, and according to the first comment in [this answer](https://stackoverflow.com/a/35373444/1127485) Jackson should convert this automatically on deserialization. Similar for other cases below.

Reply:
And according the second comment it does not work if the 0 or 1 is coming as a String, which is the case here :)

Nevertheless this is a good idea to convert it to Boolean hence I will use this JsonSerializer hint.

Reply:
done